cln-grpc/src/convert.rs

```diff
@@ -618,6 +618,8 @@ impl From<&responses::ListnodesNodes> for pb::ListnodesNodes {
             color: c.color.as_ref().map(|v| hex::decode(&v).unwrap()), // Rule #2 for type hex?
             features: c.features.as_ref().map(|v| hex::decode(&v).unwrap()), // Rule #2 for type hex?
             addresses: c.addresses.as_ref().map(|arr| arr.iter().map(|i| i.into()).collect()).unwrap_or(vec![]), // Rule #3 
+            // Also need codegen'd
+            // option_will_fund: c.option_will_fund.<I'm not certain what goes here>
         }
```

Comment:
Thanks @vincenzopalazzo - the main difficulty I'm having is that though we have affordances for the option_will_fund fields (the data structures), I cannot get the codegen process to actually put the option_will_fund onto the ListnodeNode response objects.

Reply:
I think this is a bug of the `msggen` module because the option should be inside the schema https://github.com/ElementsProject/lightning/blob/74ddc154350ee447841d009de2556b5b6bc11cfb/doc/schemas/listnodes.schema.json#L149-L156 and the `msggen` should translate this as optional parameter, but not familiar on how the msggen works so, maybe I'm wrong